### ISSUE TITLE
consine with restart scheduler not working properly

### ISSUE THREAD

**Issue by zhuliyi0** [bmaltais/kohya_ss#665]
I cannot find a parameter for how many times to repeat the scheduler, and tensor board shows the learning rate was jumping up and down at a high frequency.

**Comment by zhuliyi0** [comment]
I wonder is it one repeat per epoch? My training session had many epochs.

**Comment by zhuliyi0** [comment]
I was idiot, it is right there under advanced configurations.
